conf/messages.en

```diff
@@ -202,14 +204,16 @@ fullResultsPage.taxableProfit = Taxable Profit
 fullResultsPage.taxableProfit.daysAllocated = Days allocated to each financial year
 fullResultsPage.taxableProfit.distributions = Distributions
 fullResultsPage.taxableProfit.profitAndDistributions = Taxable profit including distributions
+fullResultsPage.calculationTableSummary = This is a table describing your Marginal Relief calculation in detail.
+fullResultsPage.tableSummary = This is a table describing your taxable profit calculation in detail.
 
 
 associatedCompanies.title = Number of associated companies
 associatedCompanies.heading = Did your company have any active associated companies?
 associatedCompanies.heading.between = Did your company have any active associated companies between {0} and {1}?
 associatedCompanies.warning = Do not include the company you''re currently calculating Marginal Relief for in your answer. Doing so will provide you with an incorrect result.
 associatedCompanies.detailsTitle = What are associated companies?
-associatedCompanies.detailsHtml = <p>A company is considered associated to another if:</p><ul class="govuk-list govuk-list--bullet"><li>one of the companies has <a href="https://www.gov.uk/hmrc-internal-manuals/company-taxation-manual/ctm60210" class="govuk-link" target="_blank">control (opens in new tab)</a> of the other</li><li>both companies are controlled by the same companies or people</li></ul>
+associatedCompanies.detailsHtml = <p>A company is considered associated to another if:</p><ul class="govuk-list govuk-list--bullet"><li>one of the companies has control of the other</li><li>both companies are controlled by the same companies or people</li></ul>
```

Comment:
Is this a mistake?

Reply:
No this is from the changes in access ticket, one of the tasks was to remove the link, but i also removed the word "control", so just adding that back in